### PR TITLE
Trinotate: fix sqlite database url

### PR DESCRIPTION
**Pull request by loraine-gueguen**
The quick fix for https://github.com/galaxyproject/tools-iuc/issues/1770

### CODE REVIEW

File: tools/trinotate/trinotate.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="trinotate" name="Trinotate" version="3.0.1.0">
+<tool id="trinotate" name="Trinotate" version="3.0.1.1">
```
Comment (nsoranzo) [inline tools/trinotate/trinotate.xml]: No need to update the tool id for a bug fix that doesn't change the tool behaviour.

Comment (loraine-gueguen) [reply tools/trinotate/trinotate.xml]: ok. Done

**Comment by bgruening** [comment]
@loraine-gueguen maybe we should cache this file ourself in https://github.com/galaxyproject/cargo-port ?

**Comment by bgruening** [comment]
Thanks for the bugfix!

**Comment by loraine-gueguen** [comment]
@bgruening I would make sense. I will have a look. Thanks

**Pull request merged**